test/domains.jl

```diff
@@ -61,3 +62,24 @@ end
     @test point.z_sfc == zlim[1]
     @test coordinates(point) == [zlim[1]]
 end
+
+
+@testset "SingleColumnLSM Domain" begin
+    domain = SingleColumnLSM(FT, zlim = zlim, nelements = nelements[3])
+
+    point = domain.surface
+    column = domain.subsurface
+
+    @test typeof(column) == Column{FT}
+    @test typeof(point) == Point{FT}
+    @test point.z_sfc == zlim[2]
```

Comment:
Lines 75, 76 I think are redundant since 74 checks that point is a `Point` domain, and the `Point` domain tests check 75, 76.

Reply:
Same for L80-83 for `Column`